documentation/src/test/java/example/ExternalMethodSourceTestDemo.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+// tag::external_MethodSource_example[]
```

Comment:
Please move the _tag_ declaration below the `package` declaration. See comments in PR.